Cargo.toml

```diff
@@ -8,6 +8,10 @@ license = "MIT OR Apache-2.0"
 readme = "README.md"
 repository = "https://github.com/Srinivasa314/alcro"
 
+[[test]]
+name = "integration_test"
+required-features = ["tokio"]
```

Comment:
```suggestion
```
Tokio shouldnt be required for all tests

Reply:
Think this is from an older version?